leakcanary-android/src/main/java/com/squareup/leakcanary/internal/DisplayLeakActivity.java

```diff
@@ -216,7 +216,19 @@ void shareHeapDump() {
     startActivity(Intent.createChooser(intent, getString(R.string.leak_canary_share_with)));
   }
 
-  void updateUi() {
+  private void invalidateOptionsMenuIfAvailable() {
```

Comment:
nitpick but name again, ```invalidateOptionsMenu``` should be sufficient, no reason to have implementation details of method in method name.